recipes/cec2/baseline/exp/exp/si.csv

```diff
@@ -0,0 +1,151 @@
+scene,listener,haspi

```

Comment:
This file exp/exp/si.csv shouldn't be included, there is already an exp/si.csv (worrying though because the numbers are different)